clang/lib/Headers/intrin.h

```diff
@@ -413,6 +413,7 @@ static __inline__ void __DEFAULT_FN_ATTRS
 __writecr3(unsigned __INTPTR_TYPE__ __cr3_val) {
   __asm__ ("mov {%0, %%cr3|cr3, %0}" : : "r"(__cr3_val) : "memory");
 }
+#endif
```

Comment:
Nice catch. We should probably also ifdef these function declarations near the top of this file.

https://github.com/llvm/llvm-project/blob/main/clang/lib/Headers/intrin.h#L96
https://github.com/llvm/llvm-project/blob/main/clang/lib/Headers/intrin.h#L126

Reply:
That chunk of declarations near the top of the file needs a lot of work to match the actual implementation; I'd prefer to just leave it for now.